src/components/filters/component.js

```diff
@@ -314,81 +389,114 @@ class Filters extends PureComponent {
-                        />
-                      </button>
+              </div>
+            )}
+            {filters.scope === 'supply_chain' && indicator && (
+              <div className="filters-section" style={{ paddingRight: 24 }}>
+                <div className="row expanded collapse">
+                  <div
+                    className="small-12 medium-12 columns"
+                    style={{
+                      marginBottom: 48,
+                      display: 'flex',
+                      alignItems: 'start',
+                    }}
+                  >
+                    <div>
+                      <h4>
+                        {indicator.name} Desired Condition
+                      </h4>
+                      <p>(Adjust slider to change results)</p>
                     </div>
-
-                    <CustomSelect
-                      options={FOOD_OPTIONS}
-                      value={filters.food}
-                      onValueChange={(selected) => {
-                        if (selected) {
-                          this.updateFilters(selected.value, 'food');
-                          logEvent('[AQ-Food] Map', 'select food security', selected.label);
-                        }
-
-                        if (
-                          selected
-                          && filters.indicator === 'none'
-                          && !FOOD_OPTIONS.find(w => w.value === selected.value).timeline
-                        ) {
-                          this.updateFilters('absolute', 'type');
-                          this.updateFilters('baseline', 'year');
-                        }
-                      }}
+                    <button
+                      style={{ marginLeft: 8 }}
+                      type="button"
+                      className="icon-container"
+                      onClick={() => this.openModal('desired-condition-thresholds')}
+                    >
+                      <Icon
+                        name="icon-question"
+                        className="-info -primary"
+                      />
+                    </button>
+                  </div>
+                  <div className="small-12 medium-12 columns">
+                    <ThresholdSlider
+                      threshold={parseFloat(filters.threshold) || 0}
+                      onChange={threshold => this.updateFilters(threshold, 'threshold')}
+                      values={indicator.rangeValues}
+                      defaultValue={indicator.defaultValue}
+                      unit={indicator.unit}
+                      ranges={indicator.items}
                     />
                   </div>
                 </div>
               </div>
-            </div>
-            <div className="filters-section">
-              <div className="row expanded collapse">
-                <div className="small-12 columns">
-                  {timeline}
+            )}
+            {filters.scope !== 'supply_chain' && (
```

Comment:
This is kind of a conditional soup, but I think you made the right call that it still doesn't warrant splitting out the supply chain filters into their own component.